libtransact/src/state/merkle/sql/migration/sqlite.rs

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------------
+ */
+
+use diesel::prelude::*;
+
+use crate::error::InternalError;
+
+const CREATE_LEAF_TABLE: &str = r#"
+    CREATE TABLE IF NOT EXISTS merkle_radix_leaf (
+        id INTEGER PRIMARY KEY,
+        address STRING NOT NULL,
+    data BLOB)
```

Comment:
This looks like it should be indented and the ) on a separate line (when compared to other statements here.

Reply:
Fixed

Reply:
looks like the ) still needs to be moved

Reply:
Fixed